rope/base/pyobjectsdef.py

```diff
@@ -354,6 +354,9 @@ def _eval(type_=type_, arg=arg):
             self.names[node.name] = pynames.DefinedName(pyfunction)
         self.defineds.append(pyfunction)
 
+    def _AsyncFunctionDef(self, node):
+        return self._FunctionDef(node)
```

Comment:
This looks ok.

> An async def function definition. Has the same fields as FunctionDef

http://greentreesnakes.readthedocs.io/en/latest/nodes.html?highlight=AsyncFunctionDef#async-and-await